pa_config/rca.conf

```diff
@@ -71,33 +71,18 @@
         "heap-range": [
           {
             "lower-bound": 0,
-            "upper-bound": 75,
-            "threshold": 15.0
-          },
-          {
-            "lower-bound": 76,
             "upper-bound": 80,
-            "threshold": 12.5
+            "threshold": 15.0
```

Comment:
Can we have some comments or more information in the PR's description on how these values are arrived at or more appropriate?

Reply:
Updated description, and also added in code comments.